packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapController.m

```diff
@@ -503,19 +337,23 @@ - (void)setMyLocationButtonEnabled:(BOOL)enabled {
   self.mapView.settings.myLocationButton = enabled;
 }
 
+/// Sets the map style, returing any error string as well as storing that error in `mapStyle` for
+/// later access.
 - (NSString *)setMapStyle:(NSString *)mapStyle {
```

Comment:
Is the return string needed any more since you added the `styleError` setter side-effect?

Reply:
Technically no, but I think it's easier to understand `return [self.controller setMapStyle:style];` in the Pigeon method implementation than:

```objc
[self.controller setMapStyle:style];
// This value has to be read after the call above because it's set as a side effect of that call.
return self.styleError;
```